portfolio/src/main/webapp/recipes/recipe-1.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html>
+  <div id="webpage">
+    <br></br>
+    <div id="content">
+      <head>
+        <meta charset="UTF-8">
+        <title>Miasya's Website</title>
+        <link rel="stylesheet" href="../style.css">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+        <script src="script.js"></script>
+
+        <!-- Top Navigation Menu ref: W3Schools-->
+        <div class="topnav" id="myTopnav">
+          <a href="../index.html" class="active">Home</a>
+          <a href="#login">Login</a>
+          <a href="../about.html">About</a>
+          <a href="../contact.html">Contact</a>
+      
+          <a href="javascript:void(0);" class="icon" onclick="navBar()">
+            <i class="fa fa-bars"></i>
+          </a>
+        </div>
+      </head>
+
+      <body>
+        <h1>Recipe Name</h1>
+        <img src="../images/eggrolls.jpg" alt="Food" width="80%"></img>
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum
```

Comment:
Please avoid submitting test code (or test data in code) to the main branch. This is risky as it makes it much easier to get to production this way. Either remove it before the submission or put it in a "experimental" file that stays in your branch.

Reply:
Are you referring to Lorem ipsum? If so I can populate that with actual text in an upcoming commit

Reply:
yes, "Lorem ipsum" here, but test strings in general. I think "whatever goes into master will end up in production" is a good rule of thumb for this.

Reply:
Ok got it!